### ISSUE TITLE
Clarification needed on copper thickness of inner PCB layers 

### ISSUE THREAD

**Issue by Selvakumar58** [LibreSolar/bms-c1#47]
In the PCB notes, we see the following text:

![image](https://github.com/LibreSolar/bms-c1/assets/146818538/99c707c8-ac1a-4a65-8dbc-433844395925)

However, in the Board setup, the details are as follows:

![image](https://github.com/LibreSolar/bms-c1/assets/146818538/144d526d-345b-4320-b9ba-17d02040dad8)

Here the units are in mils, but when converted to um, we get:

105 um outer layers
35 um inner layers

Which is the right one?

Regards,
Selva.

**Comment by pasrom** [comment]
Hello @Selvakumar58,

Could you confirm if you are using the latest commit? I noticed that in [PR #46 of the LibreSolar/bms-c1 repository](https://github.com/LibreSolar/bms-c1/pull/46), there doesn’t seem to be any difference. If you are not on the latest commit, it might be worth updating to see if that addresses your issue.

**Comment by Selvakumar58** [comment]
Hello @pasrom ,

We are using V0.3.3 (Commit no: b7895e0fe39fa203ef21a05e899bce7c4bfe8ab3) to build a few prototypes before transitioning to the latest commit. 

Any thoughts on the copper thicknesses for this commit?

Regards,
Selvakumar.

**Comment by pasrom** [comment]
From what I understand, @martinjaeger manufactures his boards with a 70 μm thickness for all four layers. It would be great if @martinjaeger could confirm this.

**Comment by martinjaeger** [comment]
Correct. We manufactured the first batch with 70u for inner and outer layer because the originally intended 105u were not suitble for the small pin pitch of the bq76952.

Both myself an @pasrom have tested the boards and 70u turns out to be sufficient for 100A with a heat sink.

**Comment by Selvakumar58** [comment]
Thank you
